src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -546,13 +551,12 @@ public int MaxResponseDrainSize
 
         protected override void Dispose(bool disposing)
         {
-            if (disposing && !_disposed)
+            if (!_disposed)
```

Comment:
Consider adding a comment that it's intentional we're explicitly ignoring ```disposing``` so that it doesn't erroneously get added back in the future.